sql/ddl_api.sql

```diff
@@ -308,3 +308,8 @@ BEGIN
     PERFORM _timescaledb_internal.attach_tablespace(hypertable_id, tablespace);
 END
 $BODY$;
+
```

Comment:
Nit. Remove whitespace? Won't `pgindent` fix this? Maybe set your editor to clean this up on save?